### PR TITLE
Zetter、Tool Kit、Corail Recycler 翻译提交

### PR DESCRIPTION
**Pull request by mamaruo**
Zetter 官译太烂（Crowdin，作者自己机翻的），对比如下

<details><summary>Details</summary>
<p>

| **键** | **英文** | **官方翻译** | **本 Pull Request** |
|:---:|:---:|:---:|:---:|
| block.zetter.artist_table | Artist Table | 艺术家的桌子 | 艺术家工作台 |
| console.zetter.result.requested_painting | Painting %1$s was requested from the sever, please wait |  | 已向服务器请求%1$s画作，请稍等 |
| console.zetter.result.exported_painting_client | Painting %1$s was saved to your Minecraft folder |  | 画作%1$s已保存至你的Minecraft文件夹 |
| console.zetter.result.exported_painting_server | Painting %1$s was saved to server folder |  | 画作%1$s已保存至服务器文件夹 |
| console.zetter.error.unknown | Unexpected error happened: %1$s |  | 出现意外错误：%1$s |
| console.zetter.error.file_write_error | Cannot write the file to save painting: %1$s |  | 无法写入文件来保存画作: %1$s |
| console.zetter.error.file_write_folder | Cannot create export folder |  | 无法创建导出文件 |
| console.zetter.error.file_write_folder_unable | Cannot create export folder |  | 无法创建导出文件夹 |
| console.zetter.error.file_write_folder_exists | File with the same name exists |  | 已存在同名文件 |
| console.zetter.error.file_write_file | Unable to write image file | 无法写入文件以保存绘画：%1$s | 无法写入图像文件 |
| console.zetter.error.invalid_painting_code | Invalid painting code: %1$s | 无效的绘画代码。%1$s | 无效的画作码：%1$s |
| console.zetter.error.painting_not_found | Painting not found: %1$s | 未找到绘画：%1$s | 未找到画作：%1$s |
| container.zetter.artistTable | Artist table | 艺术家的桌子 | 艺术家工作台 |
| container.zetter.artist_table | Artist Table | 艺术家桌 | 艺术家工作台 |
| container.zetter.artist_table.change_action | Change Action | 改变行动 | 改变行为 |
| container.zetter.artist_table.change_action.to_split | Switch to Split Mode | 切换到分割模式 | 切换到分离模式 |
| container.zetter.artist_table.combined_canvas | Painting Preview | 绘画预览 | 画作预览 |
| container.zetter.artist_table.combined_canvas.empty | No preview | 没有预览 | 预览为空 |
| container.zetter.artist_table.combined_canvas.invalid | Invalid combination | 无效组合 | 无效的组合 |
| container.zetter.artist_table.combined_canvas.loading | Loading canvases | 加载画布 | 正在加载画布 |
| container.zetter.artist_table.mode.split | Split Mode | 分割模式 | 分离模式 |
| container.zetter.painting | Painting | 绘画 | 画作 |
| container.zetter.painting.blending.additive | Additive | 添加剂 | 加上 |
| container.zetter.painting.blending.realistic | Realistic | 现实的 | 写实 |
| container.zetter.painting.blending.subtractive | Subtractive | 减法 | 减去 |
| container.zetter.painting.canvas | Canvas | 帆布 | 画布 |
| container.zetter.painting.history.undo | Undo | 解开 | 撤销 |
| container.zetter.painting.hsb | HSB Color Adjustment | HSB 颜色调整 | HSB颜色调整 |
| container.zetter.painting.palette | Palette | 调色板 | 调色盘 |
| container.zetter.painting.sliders | Adjust color (HSV) | HSV 颜色调整 | 调整颜色（HSV） |
| container.zetter.painting.sliders.hue | Hue | 色调 | 色相 |
| container.zetter.painting.sliders.intensity | Intensity | 强度 | 浓度 |
| container.zetter.painting.sliders.saturation | Saturation | 饱和 | 饱和度 |
| container.zetter.painting.sliders.size | Size | 尺寸 | 大小 |
| container.zetter.painting.tabs.bucket_parameters | Bucket parameters | 桶参数 | 颜料桶参数 |
| container.zetter.painting.tabs.inventory | Inventory | 库存 | 物品栏 |
| container.zetter.painting.tools.brush | Brush | 刷子 | 画笔 |
| container.zetter.painting.tools.bucket | Bucket Tool | 斗式工具 | 颜料桶工具 |
| container.zetter.painting.tools.eyedropper | Eyedropper | 吸管 | 取色器 |
| container.zetter.painting.tools.hand | Hand tool | 手工工具 | 手形工具 |
| entity.zetter.custom_painting_entity | Framed Painting | 裱框画 | 带框的画 |
| item.zetter.acacia_basic_frame | Acacia Frame | 金合欢画框 | 金合欢木画框 |
| item.zetter.acacia_plated_frame | Acacia Frame with Nameplate | 带铭牌的金合欢画框 | 带名牌的金合欢木画框 |
| item.zetter.birch_basic_frame | Birch Frame | 桦木画框 | 白桦木画框 |
| item.zetter.birch_plated_frame | Birch Frame with Nameplate | 带铭牌的桦木画框 | 带名牌的白桦木画框 |
| item.zetter.canvas.blank | Blank canvas | 空白的画布 | 空白画布 |
| item.zetter.canvas.painted | Painted canvas | 画布 | 画过的画布 |
| item.zetter.crimson_basic_frame | Crimson Frame | 绯红画框 | 绯红木画框 |
| item.zetter.crimson_plated_frame | Crimson Frame with Nameplate | 带铭牌的绯红画框 | 带名牌的绯红木画框 |
| item.zetter.customPaintingByAuthor | \"%1$s\" by %2$s | "%1$s" 创作者：%2$s  | %2$s创作的“%1$s” |
| item.zetter.dark_oak_plated_frame | Dark Oak Frame with Nameplate | 带铭牌的深色橡木画框 | 带名牌的深色橡木画框 |
| item.zetter.gold_basic_frame | Golden Frame | 金画框 | 金质画框 |
| item.zetter.gold_plated_frame | Golden Frame with Nameplate | 带铭牌的金画框 | 带名牌的金质画框 |
| item.zetter.iron_frame | Iron Bezel-less Frame | 铁制无边框画框 | 铁质窄边画框 |
| item.zetter.jungle_basic_frame | Jungle Frame | 丛林画框 | 丛林木画框 |
| item.zetter.jungle_plated_frame | Jungle Frame with Nameplate | 带铭牌的丛林画框 | 带名牌的丛林木画框 |
| item.zetter.mangrove_basic_frame | Mangrove Frame | 红树林框架 | 红树木画框 |
| item.zetter.mangrove_plated_frame | Mangrove Frame with Nameplate | 带铭牌的红树林框架 | 带名牌的红树木画框 |
| item.zetter.oak_plated_frame | Oak Frame with Nameplate | 带铭牌的橡木画框 | 带名牌的橡木画框 |
| item.zetter.painting | Painting | 绘画 | 画 |
| item.zetter.painting.generation.0 | Original | 原创 | 原作 |
| item.zetter.painting.generation.1 | Copy | 复制 | 副本 |
| item.zetter.painting.generation.2 | Copy of copy | 拷贝的拷贝 | 副本的副本 |
| item.zetter.painting.name_field | Name Field | 名称字段 | 名字输入框 |
| item.zetter.paints | Paints | 颜料 | Paints |
| item.zetter.palette | Palette | 调色板 | 调色盘 |
| item.zetter.spruce_basic_frame | Spruce Frame | 云杉画框 | 云杉木画框 |
| item.zetter.spruce_plated_frame | Spruce Frame with Nameplate | 带铭牌的云杉画框 | 带名牌的云杉木画框 |
| item.zetter.warped_plated_frame | Warped Frame with Nameplate | 带铭牌的诡异木画框 | 带名牌的诡异木画框 |
| painting.zetter.banned | This painting was banned by the server admins, alternative used |  | 这幅画已被服务器管理员禁用，现在使用的是其替代品 |
| forge.configgui.zetter.painting.resolution | Painting Resolution |  | 画作分辨率 |
| forge.configgui.zetter.gui.helpButton | Show Help Button |  | 显示“帮助”按钮 |

</p>
</details> 

给 Zetter 拟一个名字？

- [Corail Recycler - Minecraft Mods - CurseForge](https://www.curseforge.com/minecraft/mc-mods/corail-recycler)
- [Tool Kit - Minecraft Mods - CurseForge](https://www.curseforge.com/minecraft/mc-mods/tool-kit)
- [Zetter — Painting Mod - Minecraft Mods - CurseForge](https://www.curseforge.com/minecraft/mc-mods/zetter)

1

- [x] 我已**仔细**阅读贡献指南 [CONTRIBUTING](https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/blob/main/CONTRIBUTING.md)；
- [x] 我已确认英文原文（如 en_us.json）存在且完整，内容与中文对应；
- [x] 我已阅读并同意按 [CC BY-NC-SA 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/deed.zh) 协议发布我的作品；
- [ ] 刷新 PR 的标签/状态，有需要再点击；

### CODE REVIEW

File: projects/1.19/assets/zetter/zetter/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,105 @@
+{
+  "block.zetter.artist_table": "艺术家工作台",
+  "console.zetter.result.requested_painting": "已向服务器请求%1$s画作，请稍等",
+  "console.zetter.result.exported_painting_client": "画作%1$s已保存至你的Minecraft文件夹",
+  "console.zetter.result.exported_painting_server": "画作%1$s已保存至服务器文件夹",
+  "console.zetter.error.unknown": "出现意外错误：%1$s",
+  "console.zetter.error.file_write_error": "无法写入文件来保存画作: %1$s",
```
Comment (ChuijkYahus) [inline projects/1.19/assets/zetter/zetter/lang/zh_cn.json:7]:
```suggestion
  "console.zetter.error.file_write_error": "无法写入文件来保存画作：%1$s",
```

File: projects/1.19/assets/corail-recycler/corail_recycler/lang/zh_cn.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    "corail_recycler.item.diamond_disk": "钻石圆片",
+    "corail_recycler.item.netherite_disk": "下界合金圆片",
+    "corail_recycler.item.disk.desc": "回收器需要这个圆片才能工作",
+    "corail_recycler.message.add_recipe_success": "已添加配方",
+    "corail_recycler.message.add_recipe_failed": "添加配方失败",
+    "corail_recycler.message.export_success": "探索成功",
+    "corail_recycler.message.export_failed": "探索失败",
+    "corail_recycler.message.remove_recipe_success": "已移除配方",
+    "corail_recycler.message.remove_recipe_failed": "移除配方失败",
+    "corail_recycler.message.found_recipes": "%d recipes found",
```
Comment (ChuijkYahus) [inline projects/1.19/assets/corail-recycler/corail_recycler/lang/zh_cn.json:19]: 诶这里是没翻吗

Comment (mamaruo) [reply projects/1.19/assets/corail-recycler/corail_recycler/lang/zh_cn.json:19]:
```suggestion
    "corail_recycler.message.found_recipes": "已找到%d个配方",
```

Comment (ChuijkYahus) [reply projects/1.19/assets/corail-recycler/corail_recycler/lang/zh_cn.json:19]: 是文件里还好多没翻啊（）

**Review by ChuijkYahus — APPROVED** [review]

**Review by ChuijkYahus — APPROVED** [review]

**Pull request merged**